content/posts/survey-01/index.md

```diff
@@ -263,31 +241,27 @@ IIiIIIIIIIIIIIIIIIiIIIIIIII
 - Collection of “Standard crates” with some form of WG support
 - Push for stabilisation
 
-### Faster iteration times
-IIIIIIIIIIIIIIIII
+### Faster iteration times (17)
 
 - Hot code reloading
 - Scripting
 - Faster compile times
 - More libraries like Processing
 
 
-### Integration with existing toolchains
-IIIIIIIIIIIIII
+### Integration with existing toolchains (14)
 
 - C/C++
 - Xcode, Android studio, Visual studio, Blender
 - Unity/Unreal/Godot
 
-### Custom allocation story
-IIIIIIIIII
+### Custom allocation story (10)
 
-### Debugging support and IDE integration
-IIIII
+### Debugging support and IDE integration (5)
 
-### Interoperability between libraries
-IIIII
+### Interoperability between libraries (5)
 
 ## Can you name some libraries you are thankful for or take special interest in?
-
-[graphic pending]
+Just for fun, we asked respondents to name libraries or projects they were thankful for. In the 6 months that have passed since this survey went out there's been quite a few newcomers on the rise that would probably show more prominently here if we asked again today. Rest assured there will be more surveys.
+![Shoutouts to ecosystem projects](survey-thanks.png)
+_Thanks so much to the survey respondents and all the people out there working tirelessly make Rust a first-tier language and ecosystem for game development._
```

Comment:
```suggestion
_Thank you so much to the survey respondents and all the people out there working tirelessly to make Rust a first-tier language and ecosystem for game development._
```

Reply:
Has this been applied? I haven't used this GH feature before so I'm not quite sure what happened when I committed your first set of suggestions. These other suggestions are apparently "outdated" now.

Reply:
The second suggestion seems not to have been applied.

Reply:
Ah, I guess I should’ve opted for “add suggestion to batch” instead. Could you please resubmit the missing suggestions?